### ISSUE TITLE
unable to import WSIReader from monai.wsi_reader

### ISSUE THREAD

**Issue by a-parida12** [Project-MONAI/MONAI#6260]
**Describe the bug**
In correct deprecation warning. 

When i load a `TIFF` image using
```python
from monai.data.image_reader import WSIReader  
LoadImageD(keys=['image'], reader=WSIReader, backend='cucim', level=3, image_only=True)
```
I get the following deprecation warning 
```bash
lib/python3.11/site-packages/monai/utils/deprecate_utils.py:111: FutureWarning: <class 'monai.data.image_reader.WSIReader'>: Class `WSIReader` has been deprecated since version 0.8. use `monai.wsi_reader.WSIReader` instead.
  warn_deprecated(obj, msg, warning_category)
```
but I am unable to import `WSIReader` from `monai.wsi_reader`.

**Comment by KumoLiu** [comment]
Hi @a-parida12, thanks for pointing this out, I think there was a typo, it should be "from monai.data.wsi_reader import WSIReader".

Hi @wyli, do we need to update the deprecated message, or we can directly remove the "monai.data.image_reader. WSIReader", it was deprecated since 0.8. cc @drbeh.

**Comment by drbeh** [comment]
That's right! Sorry for the typo. I will remove the deprecated WSIReader soon after checking on some repos that depend on that.
```
from monai.data.wsi_reader import WSIReader
```

**Comment by Nic-Ma** [comment]
I think we can remove the deprecated API now as we defined the policy:
https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#backwards-compatibility

Thanks.

**Comment by drbeh** [comment]
@Nic-Ma sure, let me check on the dependent repos and will remove it as soon as possible.
